### PR TITLE
Infographic color & comparison key objects

### PR DESCRIPTION
**Pull request by benghaziboy**
@kand @MichaelButkovic @mparent61

### CODE REVIEW

File: bulbs/infographics/validators.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            assert not flags, "flags must be empty if regex is passed pre-compiled"
+            return regex
+    return SimpleLazyObject(_compile)
+
+
+@deconstructible
+class ColorValidator(object):
+    message = ('Enter a valid hexadecimal color.')
+    code = 'invalid'
+    color_regex = _lazy_re_compile(
```
Comment (mparent61) [inline bulbs/infographics/validators.py]: uncompiled regexes are still pretty fast in python, not sure if it's worth the extra code to pre-compile -- your call.

**Comment by mparent61** [comment]
Otherwise LGTM

**Comment by MichaelButkovic** [comment]
lgtm

**Comment by kand** [comment]
👍

**Comment by benghaziboy** [comment]
@mparent61 @kand a couple of updates

**Comment by mparent61** [comment]
👍

**Comment by kand** [comment]
👍

**Pull request merged**